commit 7a094a7351a20ecf845bb4c5ad9d1f4191919baf
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Tue May 29 10:52:11 2018 +0100

    overlays: Add sdtweak features for network booting
    
    It has been observed that a Pi with no SD card will poll the interface
    continuously, using up to 10% CPU. Add some new parameters to the
    sdtweak overlay to control this behaviour:
    
    poll_once   Only look for a card once, at boot time. If none is found
                then the interface is effectively disabled.
    
    enable      Set to "off" or "no" to completely disable the interface.
    
    See: https://github.com/raspberrypi/linux/issues/2567
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/overlays/README b/arch/arm/boot/dts/overlays/README
index f3ec2b6d2a4f..a9e56efd6532 100644
--- a/arch/arm/boot/dts/overlays/README
+++ b/arch/arm/boot/dts/overlays/README
@@ -1618,6 +1618,16 @@ Params: overclock_50            Clock (in MHz) to use when the MMC framework
 
         debug                   Enable debug output (default off)
 
+        poll_once               Looks for a card once after booting. Useful
+                                for network booting scenarios to avoid the
+                                overhead of continuous polling. N.B. Using
+                                this option restricts the system to using a
+                                single card per boot (or none at all).
+                                (default off)
+
+        enable                  Set to off to completely disable the interface
+                                (default on)
+
 
 Name:   smi
 Info:   Enables the Secondary Memory Interface peripheral. Uses GPIOs 2-25!
diff --git a/arch/arm/boot/dts/overlays/sdtweak-overlay.dts b/arch/arm/boot/dts/overlays/sdtweak-overlay.dts
index e4a4677f6b20..5880e7cd5d1c 100644
--- a/arch/arm/boot/dts/overlays/sdtweak-overlay.dts
+++ b/arch/arm/boot/dts/overlays/sdtweak-overlay.dts
@@ -19,5 +19,7 @@
 		force_pio        = <&frag0>,"brcm,force-pio?";
 		pio_limit        = <&frag0>,"brcm,pio-limit:0";
 		debug            = <&frag0>,"brcm,debug?";
+		enable           = <&frag0>,"status";
+		poll_once        = <&frag0>,"non-removable?";
 	};
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1174_7a094a7351a20ecf845bb4c5ad9d1f4191919baf.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1174_7a094a7351a20ecf845bb4c5ad9d1f4191919baf.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
